File: pset1/Greedy.c
#include<stdio.h>
#include<cs50.h>
int main()
{
    float chnge;
    int count=0;
    float quarter=0.25;
    float dime=0.10;
    float nickel=0.05;
   float penny=0.010;
    printf("O hai! How much change is owed?");
    chnge=get_float();
    
    while(chnge<0)
        { 
            printf("How much change is owed?\n");
            chnge=get_float(); 
             
        }
    while(chnge>=quarter)
    {
        chnge=chnge-quarter;
        count++;
    }
     while(chnge>=dime)
    {
        chnge=chnge-dime;
        count++;
    }
     while(chnge>=nickel)
    {
        chnge=chnge-nickel;
        count++;
    }
     while(chnge>=penny || chnge>=0)
    {
        
        chnge=chnge-penny;
        count++;
    }
    
    printf("\n%d\n",count);
    return 0;
}

File: pset6/greedy.py
import cs50

count=0;
quarter=25
dim=10
nickel=5
penny=1
print('O hai! How much change is owed?')
change=cs50.get_float()
amount=change*100
while amount<0:
             print("How much change is owed?\n")
             change=cs50.get_float(); 
            
while amount>=quarter:
    amount=amount-quarter
    count=count+1
    
while amount>=dim:
    amount=amount-dim
    count=count+1
    
while amount>=nickel:
    amount=amount-nickel
    count=count+1
    
while amount>=penny or amount>0:
    amount=amount-penny
    count=count+1
    
print(count)

    